frameworks/elastic/universe/resource.json

```diff
@@ -6,7 +6,7 @@
       "executor-zip": "{{artifact-dir}}/executor.zip",
       "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
       "scheduler-zip": "{{artifact-dir}}/elastic-scheduler.zip",
-      "statsd-plugin-zip": "https://github.com/mesosphere/elasticsearch-statsd-plugin/releases/download/{{elastic-version}}.0/elasticsearch-statsd-{{elastic-version}}.0.zip",
+      "statsd-plugin-zip": "https://github.com/mpereira/elasticsearch-statsd-plugin/releases/download/{{elastic-version}}.0/elasticsearch-statsd-{{elastic-version}}.0.zip",
```

Comment:
Just remember that this should not be merged like this.

Reply:
Yes, I'll upload the artifact to our repo and revert this. Still need to see if upstream is cool with the change too...